virttest/utils_disk.py

```diff
@@ -146,7 +146,7 @@ def umount(src, dst, fstype=None, verbose=False, session=None):
 
     :return: if unmounted return True else return False
     """
-    mounted = is_mount(src, dst, fstype, verbose=verbose, session=session)
+    mounted = is_mount(src, dst, fstype, verbose=True, session=session)
```

Comment:
hi @hellohellenmao , value of the verbose here is passed by the umount, so why not calling umount with verbose=True in your test case? Then you can get the same result.

Btw, does that matters if the umount function prints details?

Reply:
Yes, I need it for debugging, and if the value is False, when umounting failed I just do not know the mounted details which is needed for my debugging, I need to do it manually to login to the guest to check the details.

Reply:
Hi @hellohellenmao 
I did not get your point, I think that  you could get the details by the way @zhencliu mentioned, but we set the `verbose=True` mandatory here. 
From my understanding, your solution is the same to pass the` verbose=True` with calling `umount(src, dst, fstype=None, verbose=False, session=None)`

Reply:
Sorry that mixing you up, acctually I agree that to pass  verbose=True with calling umount, I am just keeping on failing with `ssh: connect to host github.com port 22: Connection timed out` while pushing the update.
And my last comment to zhenchao is for the answer to "Btw, does that matters if the umount function prints details?"